service-front/web/main.scss

```
@@ -12,3 +12,33 @@ $govuk-assets-path: "../assets/";
 .moj-banner--date {
   max-width: 150px;
 }
+
+.govuk-inset-text-blue {
+  font-family: "GDS Transport", arial, sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  font-weight: 400;
+  font-size: 1rem;
+  line-height: 1.25;
+  color: #0b0c0c;
+  padding: 15px;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  clear: both;
+  border-left: 2px solid #1d70b8;
+  background-color: #f3f2f1;
+}
+
+.govuk-main-wrapper-custom {
+  display: block;
+  padding-bottom: 40px;
+}
+
+.govuk-text-gray {
+  color: #505A5F;
+  font-family: "GDS Transport", arial, sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 19px;
+  line-height: 25px;
+}
```

This is a confusing class name because it's doing two things. You could shrink this down to just cover the colour change and do the rest with `.govuk-body` (e.g. `<div class="govuk-body app-text-gray">`).
```suggestion
.app-text-gray {
  color:  $govuk-secondary-text-colour;
}
```